libs/commons-database/src/main/kotlin/gov/cdc/ocio/database/dynamo/DynamoRepository.kt

```diff
@@ -0,0 +1,87 @@
+package gov.cdc.ocio.database.dynamo
+
+import com.google.gson.Gson
+import gov.cdc.ocio.database.persistence.Collection
+import gov.cdc.ocio.database.models.Report
+import gov.cdc.ocio.database.models.ReportDeadLetter
+import gov.cdc.ocio.database.persistence.ProcessingStatusRepository
+import software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient
+import software.amazon.awssdk.protocols.jsoncore.JsonNode
+
+
+/**
+ * DynamoDB repository implementation
+ *
```

Comment:
`@param` statement missing for tablePrefix in `Kdoc` comments

Reply:
Fixed